### PR TITLE
rename packages

### PR DESCRIPTION
**Pull request by anew**
also: 
- rebase to 6.0.0-snapshot and Spark 2.1.3
- clean up pom.xml

Note that release/1.0 was never merged back to develop. Thus, created release/1.1 from release/1.0 and basing this PR on that. Will merge to develop after this is merged.

### CODE REVIEW

**Comment by anwar6953** [comment]
Should the base branch of this be something other than release/1.0, since release/1.0 seems to be the branch compatible with older versions?

**Comment by anew** [comment]
Good catch. I created the branch off of release/1.1, but github still defaulted to 1.0 as the base. Changed it now.

File: pom.xml [hunk 1]
```diff
@@ -96,37 +93,20 @@
...
   <dependencies>
     <dependency>
-      <groupId>co.cask.cdap</groupId>
+      <groupId>io.cdap.cdap</groupId>
       <artifactId>cdap-etl-api</artifactId>
       <version>${cdap.version}</version>
       <scope>provided</scope>
     </dependency>
     <dependency>
-      <groupId>co.cask.cdap</groupId>
```
Comment (anwar6953) [inline pom.xml:105]: so many dependencies that weren't actually needed...?

**Review by anwar6953 — APPROVED** [review]

File: pom.xml (hunk 1 above)
Comment (anew) [reply pom.xml:105]: yeah must have been copy&paste

**Pull request merged**